README.md

```diff
@@ -208,7 +208,7 @@ to be used in production. The next steps we plan to work on are:
   [overriding proposal](https://github.com/tweag/nickel/blob/9fd6e436c0db8f101d4eb26cf97c4993357a7c38/rfcs/001-overriding.md))
 - Performance improvements
 
-## Related projects and inspirations
+## Comparison
```

Comment:
Mmh, I don't know about this change. Comparison is shorter but also carries much less information: comparison between what and what? Nix and Bazel? The different way of building Nickel, with Nix and with Cargo? Comparison of performances with and without contracts? Etc.

Reply:
"Comparison with alternatives"?

Reply:
Basically I think "comparison" is important because many users when considering a new tech, want a comparison with other, familiar tech. I think "Related projects and inspirations" does not quite capture that.

I'll take any suggestion though, as long as we help readers find what they are looking for

Reply:
I see, then Comparison with alternatives sounds better, yes. Comparison with related projects, Comparison with alternative projects, Comparison with other configuration languages?

Reply:
I went with "Comparison with other configuration languages":
* "projects" seems weird for users: users don't want to know if JSON or Dhall are projects, they are only interested in the end product, a configuration language
* repeating "configuration languages" is a nice wrap-up to remind what is Nickel, and echoes to the 1st line of the README

Done in 47bbb38b3a5201aaa3a22816339c5e78c767dce0